components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/impl/ActionExecutorServiceImpl.java

```diff
@@ -123,20 +126,24 @@ public ActionExecutionStatus<?> execute(ActionType actionType, Map<String, Objec
     }
 
     /**
-     * Resolve the actions by given the action id list and execute them.
+     * Resolve the action by given the action id list and execute them.
```

Comment:
This comment should now be updated to reflect that it is about executing one action